### PR TITLE
[CI:DOCS] Migrate podman container image

### PR DESCRIPTION
**Pull request by cevich**
Moved to: https://github.com/containers/image_build

This is needed to consolidate both the content and the automation into one location.  Having both in the same place is intended to both protect push secrets and allow for PR-level CI testing.

#### Does this PR introduce a user-facing change?

```release-note
None
```

### CODE REVIEW

File: contrib/podmanimage/README.md [hunk 1]
```diff
@@ -1,88 +1,2 @@
...
-**Note:** If you encounter a `fuse: device not found` error when running the container image, it is likely that
-the fuse kernel module has not been loaded on your host system.  Use the command `modprobe fuse` to load the
-module and then run the container image.  To enable this automatically at boot time, you can add a configuration
-file to `/etc/modules.load.d`.  See `man modules-load.d` for more details.
-
-### Blog Post with Details
-
-Dan Walsh wrote a blog post on the [Enable Sysadmin](https://www.redhat.com/sysadmin/) site titled [How to use Podman inside of a container](https://www.redhat.com/sysadmin/podman-inside-container).  In it, he details how to use these images as a rootful and as a rootless user.  Please refer to this blog for more detailed information.
+The podman container image build context and automation have been
+moved to [https://github.com/containers/image_build](https://github.com/containers/image_build)
```
Comment (edsantiago) [inline contrib/podmanimage/README.md:2]: LGTM but it might've been friendlier to link directly into the `/podman` subdirectory

Comment (cevich) [reply contrib/podmanimage/README.md:2]: That's not a bad idea, I can do that.

**Comment by cevich** [comment]
This is ready to go, new [build automation is functional and pushing images from `containers/image_build`](https://cirrus-ci.com/build/5993980085665792).

**Comment by cevich** [comment]
@mheon @baude @TomSweeneyRedHat PTAL, can we merge this please?

**Review by Luap99 — COMMENTED** [review]
/lgtm

**Pull request merged**